examples/awa-lwm2m/README.md

```diff
@@ -0,0 +1,92 @@
+# AwaBootstrap
+
+This app is an LWM2M client using Contiki with TinyDTLS.
+
+By default it will build for a Mikro-Electronica Clicker board. To
+build that, you'll need to ensure your build host has the relevant
+tools installed. This is usually built from an ubuntu host.
+
+## Running under Docker
+
+The app can also be built to run on a linux host, and can be built
+and run under Docker.  The only requirement for this is to have
```

Comment:
Not sure whether we want to add dockerized environment, .vscode related files into generic contiki examples. Generally all these examples should be built on linux host machine, with all the supported TARGETS. if there is something TARGET specific example then it should be moved to platforms/$(TARGET)/apps.

CI or dockerized test setup may be sit in a different repo but not in contiki examples. It needs a design discussion if you want to push inside the contiki examples.